dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/constants/Constants.java

```diff
@@ -850,4 +850,9 @@ private Constants() {
     public static final String REMOTE_LOGGING_GCS_CREDENTIAL = "remote.logging.google.cloud.storage.credential";
 
     public static final String REMOTE_LOGGING_GCS_BUCKET_NAME = "remote.logging.google.cloud.storage.bucket.name";
+
+    /**
+     * data quality
+     */
+    public static final String DATABASES_QUERY = "show databases";
```

Comment:
postgresql seem not work with this statement

Reply:
You are right, will update